compiler/plugins/target/AMD-AIE/iree-amd-aie/IR/CMakeLists.txt

```diff
@@ -83,6 +84,8 @@ iree_tablegen_library(
     -gen-typedef-defs AMDAIETypes.cpp.inc
 )
 
+list(APPEND IREE_COMPILER_TABLEGEN_INCLUDE_DIRS
```

Comment:
Is this needed?

Reply:
yes - without this tablegen won't find `include "iree-amd-aie/aie_runtime/AMDAIEEnums.td"`